apps/src/code-studio/pd/application_dashboard/constants.js

```diff
@@ -80,15 +80,22 @@ export const ALL_PARTNERS_VALUE = "all";
 export const UNMATCHED_PARTNER_LABEL = "No Partner/Unmatched";
 export const UNMATCHED_PARTNER_VALUE = "none";
 
-export const ALL_PARTNERS_OPTION = {label: ALL_PARTNERS_LABEL, value: ALL_PARTNERS_VALUE};
-export const UNMATCHED_PARTNER_OPTION = {label: UNMATCHED_PARTNER_LABEL, value: UNMATCHED_PARTNER_VALUE};
+export const WorkshopTypes = {
+  TEACHERCON: "teachercon",
+  LOCAL_SUMMER: "local summer",
+  BOTH: "both"
+};
+
+export const ALL_PARTNERS_OPTION = {label: ALL_PARTNERS_LABEL, value: ALL_PARTNERS_VALUE, workshopType: WorkshopTypes.BOTH};
+export const UNMATCHED_PARTNER_OPTION = {label: UNMATCHED_PARTNER_LABEL, value: UNMATCHED_PARTNER_VALUE, workshopType: WorkshopTypes.BOTH};
 
 export const RegionalPartnerValuePropType = PropTypes.oneOfType([
   PropTypes.number, // regional partner id
-  PropTypes.oneOf([ALL_PARTNERS_VALUE, UNMATCHED_PARTNER_VALUE])
+  PropTypes.oneOf([ALL_PARTNERS_VALUE, UNMATCHED_PARTNER_VALUE]),
 ]);
 
 export const RegionalPartnerPropType = PropTypes.shape({
   value: RegionalPartnerValuePropType.isRequired,
-  label: PropTypes.string.isRequired
+  label: PropTypes.string.isRequired,
+  workshopType: PropTypes.string
```

Comment:
Does this work? I would expect when we pass this to the select control it only pays attention to label and value, and workshopType will be lost. If we want this to be returned as part of the selection I think it needs to be encapsulated in the `value`. Where is this used?

Reply:
this isn't actually getting used - I'll remove.